userspace/libsinsp/parsers.cpp

```diff
@@ -4201,6 +4201,8 @@ void sinsp_parser::parse_rw_exit(sinsp_evt *evt)
 						char error[SCAP_LASTERR_SIZE];
 						scap_threadinfo scap_tinfo {};
 
+						memset(&scap_tinfo, 0, sizeof(scap_tinfo));
```

Comment:
the `scap_threadinfo scap_tinfo {};` two lines above should do almost the same. there is a reason why we are also adding the `memset` or it is just to be extra safe?